cmd/modelcmd/credentials.go

```diff
@@ -21,26 +21,60 @@ var (
 	ErrMultipleCredentials = errors.New("more than one credential detected")
 )
 
+// GetCredentialsParams contains parameters for the GetCredentials function.
+type GetCredentialsParams struct {
+	// Cloud is the cloud definition.
+	Cloud cloud.Cloud
+
+	// CloudName is the name of the cloud for which credentials are being
+	// obtained.
+	CloudName string
+
+	// CloudRegion is the name of the region that the user has specified.
+	// If this is empty, then GetCredentials will determine the default
```

Comment:
Could we please be a bit more precise of what "default" means here? Maybe even say that it's the first region in a list that is sorted in alphabetical order? :D